test/index.js

```diff
@@ -1,245 +0,0 @@
-require('should');
-var Limiter = require('..');
```

Comment:
If you are ok with it, I'll add back those tests, modified to call `newHit` instead of `get` and with some try/catch to get better error reports from mocha